pkg/resources/management.cattle.io/v3/fleetworkspace/validator.go

```diff
@@ -84,25 +87,21 @@ func (m *Mutator) Admit(request *admission.Request) (*admissionv1.AdmissionRespo
 
 	namespace := v1.Namespace{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: fw.Name,
+			Name:   fw.Name,
+			Labels: map[string]string{k8sManagedLabel: "true"},
```

Comment:
We should use the value of `rancher` here so users who see these namespaces know that Rancher is responsible for managing their creation and deletion.
```suggestion
			Labels: map[string]string{k8sManagedLabel: "rancher"},
```

Reply:
done in https://github.com/rancher/webhook/pull/260/commits/bc3715088473c65ec63c87a6da489240e56dbaef